udata/templates/home.html

```diff
@@ -36,81 +37,77 @@ <h1 class="page-header">Titre de l'actu
+                        <li class="col-md-4 col-sm-6">
+                        {% include theme('dataset/card.html') %}
+                        </li>
+                        {% endfor %}
+                    </ul>
+                </div>
+
+            </div>
+        </div>
+        {% for reuse in current_site.settings.home_reuses %}
+        <div class="item">
+            <div class="container">
+
+                <div class="row">
+                    <div class="col-lg-12">
+                        <h1 class="page-header">
+                            {{reuse.title}}
+                            <small>{{ _('Reuse')}}</small>
                         </h1>
                     </div>
                 </div>
 
                 <div class="row">
                     <div class="col-md-7">
-                        <img class="img-responsive" src="http://placehold.it/650x300">
+                        <img class="img-responsive" src="{{reuse.image}}">
                     </div>
                     <div class="col-md-5">
-                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam viverra euismod odio, gravida pellentesque urna varius vitae. Sed dui lorem, adipiscing in adipiscing et, interdum nec metus. Mauris ultricies, justo eu convallis placerat, felis enim.</p>
+                        <p>{{reuse.description | mdstrip(300) }}</p>
 
                         <div class="row">
                             <div class="col-lg-12">
-                                <h3 class="page-header">Jeux de données utilisés</h3>
-                            </div>
-
-                            <div class="col-sm-3 col-xs-6">
-                                <a href="#">
-                                  <img class="img-responsive portfolio-item" src="http://placehold.it/300x200">
-                                </a>
-                            </div>
-
-                            <div class="col-sm-3 col-xs-6">
-                                <a href="#">
-                                    <img class="img-responsive portfolio-item" src="http://placehold.it/300x200">
-                                </a>
+                                <h3 class="page-header">{{ _('Used datasets') }}</h3>
                             </div>
 
-                            <div class="col-sm-3 col-xs-6">
-                                <a href="#">
-                                    <img class="img-responsive portfolio-item" src="http://placehold.it/300x200">
-                                </a>
-                            </div>
-
-                            <div class="col-sm-3 col-xs-6">
-                                <a href="#">
-                                    <img class="img-responsive portfolio-item" src="http://placehold.it/300x200">
-                                </a>
+                            {% for dataset in reuse.datasets[:4] %}
```

Comment:
Yep, I was wondering which one should I choose between
- no limits in view and slicing in template => allow theme to limit itself the amount of required datasets/reuses
- limits in view => theme is limited by the view itself